fog-ovirt.gemspec

```diff
@@ -30,6 +29,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler"
   spec.add_development_dependency "pry"
   spec.add_development_dependency "rake"
-  spec.add_development_dependency "rubocop", "~> 0.52"
+  spec.add_development_dependency "rubocop"
```

Comment:
Let's freeze rubocop version. It tends to change under our feet and fail CI.
If you think we would like to keep rubocop up to date, I would recommend adding a separate CI job that will test against the latest rubocop once in a while and we will monitor it.

Reply:
Fixed the version, we could use dependabot for this?

Reply:
I don't have experience with dependabot, but I am not against it. We can always disable it later :shrug: